### PR TITLE
Add documentation about subctl show brokers cmd

### LINKED ISSUE #767: Document new `subctl show brokers` command
Depends on https://github.com/submariner-io/subctl/pull/70

### PR DESCRIPTION
**Pull request by Jaanki**
Closes: https://github.com/submariner-io/submariner-website/issues/767

Depends on https://github.com/submariner-io/subctl/pull/70

Signed-off-by: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

**Review by nyechiel — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

File: src/content/operations/deployment/subctl/_index.en.md [hunk 1]
```diff
@@ -189,6 +189,12 @@ Shows information about the Submariner endpoint connections with other clusters.
 
 Shows information about the Submariner endpoints in the cluster.
 
+### `show brokers`
+
+`subctl show brokers [flags]`
```
Comment (tpantelis) [inline src/content/operations/deployment/subctl/_index.en.md:194]: Are there any flags?

File: src/content/operations/deployment/subctl/_index.en.md [hunk 2]
```diff
@@ -189,6 +189,12 @@ Shows information about the Submariner endpoint connections with other clusters.
 
 Shows information about the Submariner endpoints in the cluster.
 
+### `show brokers`
```
Comment (tpantelis) [inline src/content/operations/deployment/subctl/_index.en.md:192]: I just realized the subcommand is `brokers` but there's only one broker so perhaps we should change it to `broker`.

**Review by tpantelis — CHANGES_REQUESTED** [review]

**Comment by nyechiel** [comment]
@Jaanki can you review Tom's comments please?

File: src/content/operations/deployment/subctl/_index.en.md (hunk 1 above)
Comment (Jaanki) [reply src/content/operations/deployment/subctl/_index.en.md:194]: yes, there is a `global` `kubeconfig` flag.

```
$ subctl show brokers --help
This command shows information about the Broker in the cluster

Usage:
  subctl show brokers [flags]

Flags:
  -h, --help   help for brokers

Global Flags:
      --kubeconfig string   absolute path(s) to the kubeconfig file(s)
```

Comment (tpantelis) [reply src/content/operations/deployment/subctl/_index.en.md:194]: Ok but you don't have a section that describes them like, eg, L204 below.

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/operations/deployment/subctl/_index.en.md (hunk 1 above)
Comment (Jaanki) [reply src/content/operations/deployment/subctl/_index.en.md:194]: L204 describes the "global" flag for "all" of `show` commands. Its a common description. Even other `show` subcommands don't have flag description.

**Review by tpantelis — APPROVED** [review]

**Pull request merged**